### PR TITLE
Blog: 2.15.0

### PR DESCRIPTION
**Pull request by psiinon**
Also removed most references to the softwaresecurityproject dockerhub org.

<details><summary>Screenshot</summary>
<p>

Screenshot 2024-05-07 at 11-32-12 ZAP 2 15 0

</p>
</details>

### CODE REVIEW

File: site/content/blog/2024-05-07-zap-2-15-0/index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ZAP 2.15.0 has just been [released](/docs/desktop/releases/2.15.0/), and adds support for 
+scripts as first class scan rules, restructured desktop menu items, and more...
+
+![ZAPbot with 2.15.0 banner](images/blog-2-15-0.png)
+
+This release was made possible thanks to our biggest supporter: [Crash Override](https://crashoverride.com/?zap=web).
+
+### Scripts as First Class Scan Rules
+
+Active and passive scan rules scripts can now be treated as "first class" scan rules.
```
Comment (kingthorin) [inline site/content/blog/2024-05-07-zap-2-15-0/index.md:23]: rule scripts 
or
script rules 

They shouldn't both be plural

File: site/content/blog/2024-05-07-zap-2-15-0/index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+This means that they can be individually referenced in an 
+[active scan policy](/docs/desktop/start/features/scanpolicy/), in the 
+[passive scan rules](/docs/desktop/ui/dialogs/options/pscanrules/) page, and in
```
Comment (kingthorin) [inline site/content/blog/2024-05-07-zap-2-15-0/index.md:26]: page ➡️ options

File: site/content/blog/2024-05-07-zap-2-15-0/index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+There is a brand new ZAP GitHub action - the 
+[ZAP Automation Framework Scan](https://github.com/marketplace/actions/zap-automation-framework-scan).
+This allows you run an [Automation Framework](/docs/automate/automation-framework/) plan in a GitHub action.
+
+Why is this important?
+Because the Automation Framework provides a [great balance](/docs/getting-further/automation/automation-options/)
+between ease of use and flexibility + functionality.
+ 
+If you want to perform any non trivial automation with ZAP then the
```
Comment (kingthorin) [inline site/content/blog/2024-05-07-zap-2-15-0/index.md:53]: non-trivial

File: site/content/blog/2024-05-07-zap-2-15-0/index.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+We are still using the [softwaresecurityproject](https://hub.docker.com/u/softwaresecurityproject) org for 2.15.0 but we will probably not use it for the following releases.
+
+We do recommend that you switch from `softwaresecurityproject` to [zaproxy](https://hub.docker.com/u/zaproxy) sooner rather than later.
+
+### Support
+
+While the main funding for ZAP development is thanks to [Crash Override](https://crashoverride.com/?zap=web),
+we are continuing to work towards self sustainability.
+
+If you need ZAP support then head over to the [Support](https:///support/) page. All of the money raised via the professional services will help fund ZAP development - ZAP is a non-profit open source project.
```
Comment (kingthorin) [inline site/content/blog/2024-05-07-zap-2-15-0/index.md:73]: Link is busted

**Comment by psiinon** [comment]
Thanks - think I've fixed all of those, in this PR and in https://github.com/zaproxy/zap-core-help/pull/545

File: site/content/blog/2024-05-07-zap-2-15-0/index.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+While the main funding for ZAP development is thanks to [Crash Override](https://crashoverride.com/?zap=web),
+we are continuing to work towards self sustainability.
+
+If you need ZAP support then head over to the [Support](/support/) page. All of the money raised via the professional services will help fund ZAP development - ZAP is a non-profit open source project.
+ 
+### Dependency Updates
+
+As usual the release includes dependency updates, see the [release notes](/docs/desktop/releases/2.15.0/) for details.
+
+There are of course a large number of other enhancements and fixes which are detailed in the [release notes](/docs/desktop/releases/2.15.0/).
```
Comment (kingthorin) [inline site/content/blog/2024-05-07-zap-2-15-0/index.md:79]: Add another heading? Conclusion or something. It's weird that this follows dependency updates.

Your closing sentence is obvious but this seems out of place somehow

Comment (psiinon) [reply site/content/blog/2024-05-07-zap-2-15-0/index.md:79]: Ripped off from the 2.14 blog .. but yeah, I agree. Will try to come up with something, but more suggestions welcome..

Comment (psiinon) [reply site/content/blog/2024-05-07-zap-2-15-0/index.md:79]: Merged and renamed the last 2 sections - that look better?

Comment (kingthorin) [reply site/content/blog/2024-05-07-zap-2-15-0/index.md:79]: Did you push without committing or something? I'm not seeing it

Comment (psiinon) [reply site/content/blog/2024-05-07-zap-2-15-0/index.md]: Er, maybe not 😉 Should be there now ..

**Review by kingthorin — APPROVED** [review]
Looks good

**Comment by psiinon** [comment]
Updated to include details for setting the log levels via the cmdline and API.

**Pull request merged**